src/ui/mesh/qgsmeshtransformcoordinatesdockwidgetbase.ui

```diff
@@ -38,7 +38,14 @@
       </property>
      </widget>
     </item>
-    <item row="4" column="0" colspan="2">
+    <item row="4" column="0">
+     <widget class="QCheckBox" name="mCheckBoxZFromProjectTerrain">
+      <property name="text">
+       <string>Get Z value from project terrain</string>
```

Comment:
Wouldn't it be more proper to say:
```suggestion
       <string>Get Z value from the project terrain</string>
```
Any native speaker?

Reply:
AI tells that this is better ;-)